reference/create-reports.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Create Reports"
+api: "POST https://api.promptlayer.com/reports"
+---
+
+This endpoint allows for the creation of reports associated with test datasets. Reports are generated by analyzing the specified test dataset, which must be accessible within the user's workspace.
+
+## Example Code
+```python
+import requests
+
+response = requests.post(
+    "https://api.promptlayer.com/reports",
+    json={
+        "name": "<REPORT_NAME>",
+        "test_dataset_id": <TEST_DATASET_ID>,
+    }
+)
+```
+
+
+## Request Parameters
+
+<ParamField body="name" type="string">
+    The name of the report to be created. Must be between 1 and 255 characters in length.
+</ParamField>
+
+<ParamField body="test_dataset_id" type="integer">
+    The ID of the test dataset on which the report will be based. The dataset must be within an accessible workspace.
```

Comment:
Why call it "test dataset" to the user, that doesn't make sense because we call it Dataset everywhere.

"The ID of the dataset on which the report will be based. The dataset must be within an accessible workspace to the user."